client/src/store/categories.js

```diff
@@ -24,9 +24,14 @@ const filterCategoryIds = (isIncome) => {
     return selectedIds;
 };
 
+const getCategoryColorById = (id) => {
+    const { value } = state;
+    return id ? value[id]["color"] : undefined;
+};
+
```

Comment:
```suggestion
const getCategoryColorById = (id) => {
    const { value } = state;
    return (value[id] || {})["color"];
};
```

이렇게 하시면 원하는 로직인 것 같습니다!!